src/api/models/view_models/base.py

```diff
@@ -83,7 +89,14 @@ class FishAttributeView(FishAttribute):
                 OR fish_species.genus_id = r.attribute_id
                 OR fish_genus.family_id = r.attribute_id
             )
+            INNER JOIN fish_species_regions sr ON (fish_species.fishattribute_ptr_id = sr.fishspecies_id) 
             WHERE r.grouping_id = fish_grouping.fishattribute_ptr_id
+            AND sr.region_id IN (
+                SELECT region.id 
```

Comment:
can this subselect be done before the overall select (`WITH`) so it's DRYed up

Reply:
Did you one better